dds_web/scheduled_tasks.py

```diff
@@ -242,8 +242,42 @@ def monthly_usage():
             scheduler.app.logger.info(usage)
 
         scheduler.app.logger.debug("Task: Projects usage from database")
+        # a mock for a unit with two projects with some usage in bhours
+        usage_data_mock = {
+            unit.safespring_name: {
+                1: {
+                    "usage": 2656548,
+                    "cost": 251.3592,
+                },
+                2: {
+                    "usage": 767686,
+                    "cost": 72.6408,
+                },
+                "Unit usage": 3424234,
+                "Unit cost": 342,
+            },
+            "unit.safespring_name2": {
+                3: {
+                    "usage": 2656548,
+                    "cost": 251.3592,
+                },
+                4: {
+                    "usage": 767686,
+                    "cost": 72.6408,
+                },
+                "Unit usage": 3424234,
+                "Unit cost": 342,
+            },
+        }
+
+        # go through units and projects in the database and
+        # construct a usage and cost data dictionary
+        usage_data = {}
         try:
             for unit in db.session.query(models.Unit).with_for_update().all():
+                usage_data[unit.safespring_name] = {}
```

Comment:
Maybe we can remove the for loop above now and use that code in this for loop instead?
```suggestion
                # Get info from safespring / sunet (replace with e.g. api call later)
                safespring_data[unit.safespring_name] = {
                    "TotalBytes": 434595434499,
                    "TotalBytesRounded": 1434614451200,
                    "TotalEntries": 10333,
                }
                usage = f"Total usage in cloud for unit {unit.name} ({unit.safespring_name}): {safespring_data[unit.safespring_name]['TotalBytes']}"
                scheduler.app.logger.info(usage)

                # Calculate usage per project and total in db
                usage_data[unit.safespring_name] = {}
```